web_app/dist/title.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+        <link rel="icon" type="images/png" href="images/icon.png" />
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="title.css">
+    </head>
+
+    <body>
+    
+        <div class="title-text-container absolute-center">
+            <h1 id="title" class="title-page-font title-page-spacing">TaskViewer</h1>
```

Comment:
Does putting the `title-page-font` class on the surrounding `div` instead work?? Another potential solution, what if we put the styles from `title-page-font` on the `title-text-container` class instead?

Reply:
first solution proposed works, fixed.